library/ssl_tls13_server.c

```diff
@@ -208,7 +208,7 @@ static int ssl_tls13_offered_psks_check_identity_match_ticket(
      * the "ticket_lifetime" value which was provided with the ticket.
      *
      */
-    if (server_age > MBEDTLS_SSL_TLS1_3_MAX_ALLOWED_TICKET_LIFETIME * 1000) {
+    if (server_age > session->ticket_lifetime * 1000) {
```

Comment:
Please update the comment above. The first part is not relevant anymore I think. Otherwise ssl_ticket.c checks for the ticket age as well but because of 
```
if (ticket_lifetime > MBEDTLS_SSL_TLS1_3_MAX_ALLOWED_TICKET_LIFETIME) {     
        ticket_lifetime = MBEDTLS_SSL_TLS1_3_MAX_ALLOWED_TICKET_LIFETIME;       
    }
```
in `ssl_tls13_write_new_session_ticket_body()` we have to do it again here. It would be better to have a comment explaining that.

Reply:
I just make two change
- In `ssl_tls13_write_new_session_ticket_body()` , raise error at this case due to consistent issue
- check lifetime here. Because above parser is a callback function, it might return any value(like our tests). So I think that statement is to check the return value from `f_ticket_parse()`